Refresh.GameServer/Refresh.GameServer.csproj

```diff
@@ -65,6 +71,7 @@
       <PackageReference Include="BCrypt.Net-Next" Version="4.0.3" />
       <PackageReference Include="CommandLineParser" Version="2.9.1" />
       <PackageReference Include="Discord.Net.Webhook" Version="3.13.0" />
+      <PackageReference Include="IronCompress" Version="1.5.1" />
```

Comment:
This library mentions a lack of support for Linux arm32 for LZO, which I believe is one of our targets? Kinda? I'm unsure if this is actually used in production anywhere but it was pushed for by @Slendy when adding Docker support so it's something to consider

Reply:
This is a valid concern, we should open an issue upstream about arm32 linux (and open a tracking issue in Refresh aswell), but given this is a non-essential feature which is disabled by default, i dont feel trying to find an ARM32 solution should be blocking for merging. If required, the server owner could install the required native library themselves too (in this case being [`minilzo`](https://github.com/yuhaoth/minilzo/tree/b088e80336ef11c5a29ec949a950d4f0fec9ec05))

Reply:
Actually, I think arm32 isn't worth chasing - if it becomes a problem then we'll handle it. As it stands, everyone ever is dropping 32-bit support apart from Raspbian who still prominently features arm32 over arm64. But I think that's about the only case.